build_debian.sh

```diff
@@ -409,14 +412,8 @@ set /files/etc/sysctl.conf/net.core.somaxconn 512
 
 " -r $FILESYSTEM_ROOT
 
-if [[ $CONFIGURED_ARCH == amd64 ]]; then
-    # Configure mcelog to log machine checks to syslog
-    sudo sed -i 's/^#syslog = yes/syslog = yes/' $FILESYSTEM_ROOT/etc/mcelog/mcelog.conf
-fi
-
-## docker-py is needed by Ansible docker module
-sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT easy_install pip
-sudo https_proxy=$https_proxy LANG=C chroot $FILESYSTEM_ROOT pip install 'docker-py==1.6.0'
+## docker is needed by Ansible docker module
```

Comment:
Is it confirmed that Ansible's "docker" module works with the "docker" Python package (rather than "docker-py")?

If so, I suggest change wording to `## docker Python API package is needed by Ansible docker module`

Reply:
yes, we have upgrade ansible to 2.8.9 which uses docker instead of docker-py